marlowe-playground-client/src/Demos/View.purs

```diff
@@ -18,12 +18,16 @@ render ::
   state ->
   ComponentHTML Action ChildSlots m
 render state =
-  div [ classes [ ClassName "projects-container" ] ]
-    [ h1_ [ text "Example Files" ]
-    , demoFile (wrap "Escrow") "Escrow" "Escrow is a financial arrangement where a third party holds and regulates payment of the funds required for two parties involved in a given transaction."
-    , demoFile (wrap "ZeroCouponBond") "Zero Coupon Bond" "A zero-coupon bond is a debt security that does not pay interest but instead trades at a deep discount, rendering a profit at maturity, when the bond is redeemed for its full face value."
-    , demoFile (wrap "CouponBondGuaranteed") "Coupon Bond Guaranteed" "A guaranteed bond is a debt security that offers a secondary guarantee that interest and principal payments will be made by a third party, should the issuer default. It can be backed by a bond insurance company, a fund or group entity, a government authority, or the corporate parents of subsidiaries or joint ventures that are issuing bonds."
-    , demoFile (wrap "Swap") "Swap" "A swap is a derivative contract through which two parties exchange the cash flows or liabilities from two different financial instruments. Most swaps involve cash flows based on a notional principal amount such as a loan or bond, although the instrument can be almost anything. Usually, the principal does not change hands. Each cash flow comprises one leg of the swap. One cash flow is generally fixed, while the other is variable and based on a benchmark interest rate, floating currency exchange rate or index price. "
+  div_
+    [ div [ classes [ ClassName "modal-header" ] ]
+        [ h2 [ classes [ ClassName "title" ] ] [ text "Demo Files" ] -- TODO: create an HTML helper so all dialogs have the same header/title?
```

Comment:
I think that would be a good idea. Or a CSS rule of `.modal-header h2 { ... }`.

OT, but in PureScript I'd strongly recommend putting TODO markers in the type signature as `Warn (Text "TODO: create an HTML helper...") =>`. Then they show up in the compilation stage and are harder to ignore.

Reply:
done